nx/source/system/readtp.s

```diff
@@ -0,0 +1,10 @@
+	.section .text.__aarch64_read_tp, "ax", %progbits
+	.global __aarch64_read_tp
+	.type __aarch64_read_tp, %function
+	.align 2
+	.cfi_startproc
+__aarch64_read_tp:
```

Comment:
Why is this in assembly

Reply:
It needs to be, __aarch64_read_tp is not following standard calling conventions (neither was __aeabi_read_tp in 32-bit ARM).

Reply:
OK